Application/EdFi.Ods.Sandbox/Provisioners/SandboxProvisionerBase.cs

```diff
@@ -11,23 +11,24 @@
 using EdFi.Admin.DataAccess.Utils;
 using EdFi.Ods.Common.Configuration;
 using EdFi.Ods.Common.Extensions;
+using Microsoft.Extensions.Configuration;
 
 namespace EdFi.Ods.Sandbox.Provisioners
 {
     public abstract class SandboxProvisionerBase : ISandboxProvisioner
     {
-        private readonly IConfigValueProvider _configValueProvider;
+        private readonly IConfiguration _configuration;
```

Comment:
same as above make an independent copy

Reply:
Created new file and wrapped inside NETFRAMEWORK